website/docs/first-composite.mdx

```diff
@@ -9,6 +9,12 @@ This page presents how to create your first composite and deploy it to your loca
 
 ## Running a local Ceramic node
 
+:::caution
+
+Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_COMPOSEDB` environment variable to `true`, before running a node.
```

Comment:
```suggestion
Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_COMPOSEDB` environment variable to `true`, before running a node.  Note that ComposeDB is not yet supported on Ceramic mainnet.
```